packages/web/src/utils/web.config.ts

```diff
@@ -84,6 +84,7 @@ function getBrowserPartial(
         deployUrl,
         sri: subresourceIntegrity,
         noModuleEntrypoints: ['polyfills-es5'],
+        publicUrl: options.publicUrl,
```

Comment:
We have `options.publicUrl` available since it's specific to CRA. Our equivalent would be `deployUrl` for assets.